app/src/main/java/com/github/polybooks/FillSaleActivity.kt

```diff
@@ -149,7 +178,7 @@ class FillSaleActivity : AppCompatActivity(), AdapterView.OnItemSelectedListener
             price = findViewById<EditText>(R.id.filled_price).text.toString().toFloat(),
             condition = bookConditionSelected!!,
             state = SaleState.ACTIVE,
-            image = null
+            image = TODO()
```

Comment:
Putting this here crashes the app, making it impossible to add a sale. Leaving it at null with a todo comment so this functionality can still be shown/used is better